solr/core/src/java/org/apache/solr/search/Grouping.java

```diff
@@ -437,32 +425,7 @@ public void execute() throws IOException {
    */
   private void searchWithTimeLimiter(final Query filterQuery, Collector collector)
       throws IOException {
-    if (cmd.getTimeAllowed() > 0) {
-      if (timeLimitingCollector == null) {
-        timeLimitingCollector =
-            new TimeLimitingCollector(
-                collector, TimeLimitingCollector.getGlobalCounter(), cmd.getTimeAllowed());
-      } else {
-        /*
-         * This is so the same timer can be used for grouping's multiple phases.
-         * We don't want to create a new TimeLimitingCollector for each phase because that would
-         * reset the timer for each phase.  If time runs out during the first phase, the
-         * second phase should timeout quickly.
-         */
-        timeLimitingCollector.setCollector(collector);
-      }
-      collector = timeLimitingCollector;
-    }
-    try {
-      searcher.search(QueryUtils.combineQueryAndFilter(query, filterQuery), collector);
-    } catch (TimeLimitingCollector.TimeExceededException
-        | ExitableDirectoryReader.ExitingReaderException x) {
-      // INFO log the (possibly quite long) query object separately
-      log.info("Query: {}; ", query);
-      // to make WARN logged exception content more visible
-      log.warn("Query: {}; ", query.getClass().getName(), x);
-      qr.setPartialResults(true);
-    }
+    searcher.search(filterQuery, collector);
```

Comment:
[Q] Is this logic moving somewhere else?  I'm not sure how/why we're nuking all of this logic?